### PR TITLE
[CPF-4327] Add Cache-Control header to ResourceImageServlet

### PR DESCRIPTION
**Pull request by KernelDeimos**
~This doesn't fix anything, but the header probably should be there at least.~ Edit: this works fine but most of us have "Disable cache" turned on all the time

### CODE REVIEW

File: src/foam/nanos/servlet/ResourceImageServlet.java [hunk 1]
```diff
@@ -49,7 +49,8 @@ protected void service(HttpServletRequest req, HttpServletResponse resp)
       String ext = EXTS.get(request.substring(request.lastIndexOf('.') +1));
       try ( BufferedInputStream bis = new BufferedInputStream(is) ) {
         resp.setContentType(!SafetyUtil.isEmpty(ext) ? ext : DEFAULT_EXT);
-        resp.setHeader("Content-Disposition", "filename=\"" + StringEscapeUtils.escapeHtml4(request) + "\"");
+        // resp.setHeader("Content-Disposition", "filename=\"" + StringEscapeUtils.escapeHtml4(request) + "\"");
```
Comment (jlhughes) [inline src/foam/nanos/servlet/ResourceImageServlet.java]: Why did you remove this, not that I remember what is was for.

Comment (KernelDeimos) [reply src/foam/nanos/servlet/ResourceImageServlet.java]: Oh, I forgot to mention that in my PR description. As far as I can tell this header is incorrect so I decided to comment it out while I was testing the OverlayActionListView icons. According to the MDN page the Content-Disposition header should begin with `inline` or `attachment` and filename should be just a name for the file if it's downloaded (right now it's a path)

Comment (kgrgreer) [reply src/foam/nanos/servlet/ResourceImageServlet.java]: Yes, the header format is incorrect, but also, I don't think it makes sense. Look at the commit for when this was introduced to see why the developer added it. If no good reason is presented, then remove in a separate PR which explains why.

Comment (jlhughes) [reply src/foam/nanos/servlet/ResourceImageServlet.java]: It was there from the inception of the class, so simply originally wrong .
@KernelDeimos just remove the line.

Comment (kgrgreer) [inline src/foam/nanos/servlet/ResourceImageServlet.java]: @KernelDeimos Please remove this line and I'll merge.
